dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2766,6 +2766,20 @@
       },
       "description": "The reference point that the ordinal number denotes."
     },
+    "RangeInclusivity": {
+      "type": "string",
+      "enum": [
+        "noneInclusive",
+        "leftInclusive",
+        "rightInclusive",
+        "leftRightInclusive"
+      ],
```

Comment:
While REST API guidelines to recommend camelCase for enums, this is not idiomatic of your service that uses PascalCase for enums. I recommend sticking with PascalCase.

Reply:
Fixed!